content/docs/tutorials/es2015.md

```diff
@@ -127,8 +127,8 @@ import CoverageBabel from './index'
 
 require('chai').should()
 
-describe('CoverageBabel', () => {
-  it('returns hello world message', () => {
+describe('CoverageBabel', function() {
+  it('returns hello world message', function() {
```

Comment:
silly nit; but mind replacing `function()` with `function ()`.

thanks for introducing me to this mocha caveat.